garage/tf/policies/epsilon_greedy_policy.py

```diff
@@ -0,0 +1,118 @@
+"""
+ϵ-greedy policy.
+
+Random exploration according to the value of epsilon.
+"""
+import numpy as np
+
+from garage.core import Serializable
+from garage.misc.overrides import overrides
+from garage.tf.policies import Policy
+from garage.tf.spaces import Box
+
+
+class EpsilonGreedyPolicy(Policy, Serializable):
+    """
+    ϵ-greedy policy.
+
+    Select action based on the value of ϵ. ϵ will decrease from
+    max_epsilon to min_epsilon within decay_ratio * total_step.
+
+    At state s, with probability
+    1 − ϵ: select action = argmax Q(s, a)
+    ϵ    : select a random action
```

Comment:
Random how? What is the distribution?

Reply:
I will change it to `np.random.uniform(...)` in choosing random actions so it will be from a uniform distribution.